Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/SortAdapter.cpp

```diff
@@ -38,6 +39,69 @@ namespace AZ::DocumentPropertyEditor
         }
     }
 
+    template <typename IteratorType>
+    void IncrementIterator(IteratorType iter, size_t distance)
+    {
+        for (size_t index = 0; index < distance; ++index)
+        {
+            ++iter;
+        }
+    }
+
+    Dom::Path RowSortAdapter::MapPath(const Dom::Path& path, bool mapToSource)
+    {
+        if (!m_sortActive)
+        {
+            return path;
+        }
+
+        Dom::Path mappedPath;
+        const auto* currNode = m_rootNode.get();
+
+        // go through each entry in the path, looking for an existing mapping at the give index
+        for (const auto& pathEntry : path)
+        {
+            if (pathEntry.IsIndex())
+            {
+                bool found = false;
+                for (auto element : AZStd::views::zip(currNode->m_indexSortedChildren, currNode->m_adapterSortedChildren))
```

Comment:
You can use a structured binding here
```suggestion
                for (auto [sortIndex, adapterIndex] : AZStd::views::zip(currNode->m_indexSortedChildren, currNode->m_adapterSortedChildren))
```

Reply:
done, thanks!